lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -38,28 +38,47 @@ class InspectorIssues extends Gatherer {
 
   /**
    * @param {LH.Gatherer.PassContext} passContext
-   * @param {LH.Gatherer.LoadData} loadData
    * @return {Promise<LH.Artifacts['InspectorIssues']>}
    */
-  async afterPass(passContext, loadData) {
+  async afterPass(passContext) {
     const driver = passContext.driver;
-    const networkRecords = loadData.networkRecords;
 
     driver.off('Audits.issueAdded', this._onIssueAdded);
     await driver.sendCommand('Audits.disable');
     const artifact = {
       /** @type {Array<LH.Crdp.Audits.MixedContentIssueDetails>} */
       mixedContent: [],
+      /** @type {Array<LH.Crdp.Audits.SameSiteCookieIssueDetails>} */
+      sameSiteCookies: [],
+      /** @type {Array<LH.Crdp.Audits.BlockedByResponseIssueDetails>} */
+      blockedByResponse: [],
+      /** @type {Array<LH.Crdp.Audits.HeavyAdIssueDetails>} */
+      heavyAds: [],
+      /** @type {Array<LH.Crdp.Audits.ContentSecurityPolicyIssueDetails>} */
+      contentSecurityPolicy: [],
     };
 
     for (const issue of this._issues) {
-      if (issue.details.mixedContentIssueDetails) {
-        const issueDetails = issue.details.mixedContentIssueDetails;
-        const issueReqId = issueDetails.request && issueDetails.request.requestId;
-        if (issueReqId &&
-          networkRecords.find(req => req.requestId === issueReqId)) {
-          artifact.mixedContent.push(issue.details.mixedContentIssueDetails);
-        }
+      switch (issue.code) {
```

Comment:
what's our expectation for new codes that we don't have listed here?

seems like something changed from the "catch-all" plan from before.

Reply:
We will catch everything, explicitly, and with the help of the test you wrote that parses the protocol.

We don't need to handle an unknown type, because Lighthouse tests should fail if there is a new, unhandled type.

Reply:
I guess there's the case where a Lighthouse release lags behind shiny new stuff in the protocol (misaligned releases). In that case, do you really want to see a row with just the issue type code?

Reply:
> the help of the test you wrote that parses the protocol

now that I've looked at the tests in this PR, I'm curious what this is referring to :)

Reply:
Paul has a github action "cron" test.

Reply:
For the scope of this PR, I'll stick to explicitly handling the issues that we're aware of. Given the structure of the Inspector Issues in both LH and CDP, handling any leftover issue types is non-trivial. I've created an issue for tracking this and will handle in a follow-up. #11530